conda/cli/main_init.py

```diff
@@ -28,12 +28,7 @@ def execute(args, parser):
         return initialize_dev(selected_shells[0])
 
     else:
-        for_user = args.user
-        if not (args.install and args.user and args.system):
-            for_user = True
-        if args.no_user:
-            for_user = False
-
+        for_user = args.user and not args.system
```

Comment:
I think this is what we want.  If `--system` is specified, we don't want to do a user init.

I still don't quite understand what the desired behavior with `--install` is.  Maybe we want to skip user init in that case, too.  If so, that isn't the behavior now.

Reply:
Since `--install` short circuits above on line 17, it serves no additional purpose. Perfectly ok to remove from the `--user/--no-user/--system` logic.